src/main/java/com/google/devtools/build/lib/remote/GrpcRemoteCache.java

```diff
@@ -154,7 +154,10 @@ public void close() {
   }
 
   public static boolean isRemoteCacheOptions(RemoteOptions options) {
-    return options.remoteCache != null;
+    // check against 'http' rather than 'grpc' because gRPC is a default protocol when none provided
+    return options.remoteCache != null
+        && !options.remoteCache.isEmpty()
```

Comment:
remove this line

Reply:
done

Reply:
Actually, here we check whether the string `!startWith`, which is true for an empty string => an empty String will be considered as enabled cache without host:port information.
Should we consider grpcCache enabled when `--remote_cache=""` or is there anything I should know?

Reply:
Right. I looked too fast - sorry :). Instead of `!options.remoteCache.toLowerCase().startsWith("http")` consider using `new URI(options.remoteCache).getScheme()` and checking for the empty string and `grpc`.

Reply:
done